pkg/loki/loki.go

```diff
@@ -42,15 +42,16 @@ func New(reg prometheus.Registerer, c Config, l log.Logger) (*Loki, error) {
 			continue
 		}
 
-		r := prometheus.WrapRegistererWith(prometheus.Labels{"loki_name": ic.Name}, reg)
+		r := prometheus.WrapRegistererWith(prometheus.Labels{"loki_config": ic.Name}, reg)
+		il := log.With(l, "loki_config", ic.Name)
```

Comment:
Sneaking in a Loki change 🤫 I forgot to add a label to the logs to indicate which config it was coming from

cc @mattdurham